lua/methods/coalitions.lua

```diff
@@ -21,13 +21,158 @@ local skill = {
   Player = 5
 }
 
+-- types for aircraft taking off
+local takeoffTypes = {
+  [1] = "TakeOff",
+  [2] = "TakeOffParking",
+  [3] = "TakeOffParkingHot",
+  [4] = "Turning Point",
+  runway = "TakeOff",
+  cold = "TakeOffParking",
+  hot = "TakeOffParkingHot",
+  air = "Turning Point"
+}
+
+-- actiions for aircraft taking off
+local takeoffActions = {
+  [1] = "From Runway",
+  [2] = "From Parking Area",
+  [3] = "From Parking Area Hot",
+  [4] = "Turning Point",
+  runway = "From Runway",
+  cold = "From Parking Area",
+  hot = "From Parking Area Hot",
+  air = "Turning Point",
+}
+
 --local altitudeType = {
 --  [1] = "BARO",
 --  [2] = "RADIO",
 --  BARO = 1,
 --  RADIO = 2
 --}
 
+local buildAirplanePoints = function(points)
+  local builtPoints = {}
+  for _, pointData in pairs(points) do
+    local pointVec3
+    if type(pointData.place) == "string" then
+      if Airbase.getByName(pointData.place) then
+        pointVec3 = Airbase.getByName(pointData.place):getPoint()
+        env.info("ab point type")
```

Comment:
Can you remove these log statements. They do not provide much actionable information and we have a separate logger for gRPC anyway. We should not be logging to the env.